_rules/html-page-lang-matches-default-ucwvc8.md

```diff
@@ -0,0 +1,227 @@
+accessibility_requirements:
+  wcag20:3.1.1: # Language of Page (A)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+  wcag-technique:H57: # Using the language attribute on the HTML element
+    forConformance: false
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: satisfied
+input_aspects:
+  - DOM tree
+  - Accessibility tree
+  - CSS Styling
+  - Language
+acknowledgments:
+  authors:
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any [document element][] if it is an `html` element for which one of the following is true:
+
+- The [document element][] has a `lang` attribute with a value that is a [valid language tag][]; and
+- The [document element][] is in a [top-level browsing context][]; and
+- The [document element][] has a [node document][] with a [content type][] of `text/html`; and
+- The [document element][] has a defined [default page language][].
+
+## Expectation
+
+For each test target, the [primary language][] of the [valid language tag][] matches the [default page language][] of the test target.
+
+## Assumptions
+
+- The way to determine the default language of a page should be done by counting the number of visible words used in each language. If the default language needs to be derived in some other way, this rule may produce incorrect results.
+
+- The language of the page can be set by other methods than the `lang` attribute, for example using HTTP headers or the `meta` element. These methods are not supported by all assistive technologies. This rule assumes that these other methods are insufficient to satisfying [Success Criterion 3.1.1: Language of Page](https://www.w3.org/TR/WCAG21/#language-of-page).
+
+- This rule assumes that user agents and assistive technologies can programmatically determine [valid language tags](#valid-language-tag) even if these do not conform to the [BCP 47][] syntax.
+
+- This rule assumes that [grandfathered tags][] are not used as these will not be recognized as [valid language tags](#valid-language-tag).
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+- [Understanding Success Criterion 3.1.1: Language of Page](https://www.w3.org/WAI/WCAG21/Understanding/language-of-page.html)
+- [H57: Using language attributes on the html element](https://www.w3.org/WAI/WCAG21/Techniques/html/H57)
+- [BCP 47: Tags for Identifying Languages](https://www.ietf.org/rfc/bcp/bcp47.txt)
+- [The `lang` and `xml:lang` attributes](https://html.spec.whatwg.org/multipage/dom.html#the-lang-and-xml:lang-attributes)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This page has a [default language][default page language] of English, which matches the language indicated in the `lang` attribute.
+
+```html
+<!DOCTYPE html>
```

Comment:
I think the DOCTYPE is not needed when its html, as it will be added automatically (https://github.com/act-rules/act-rules-web/blob/3cb73b7cb5e3a39b6c342fcb7a89ab6551dcbc32/build/create-testcases.js).

Reply:
Wow, odd, why doesn't it check if there is a doctype before adding one in? :/ I've filed an issue. I'll take them out.